src/ai/backend/account_manager/cli/start_server.py

```diff
@@ -0,0 +1,23 @@
+"""
+Lightweight entry point wrapper for the account-manager start-server command.
+
+This module provides a minimal CLI entry point that defers heavy imports
+until the command is actually executed, improving CLI startup time.
+"""
+
+from __future__ import annotations
+
+import click
+
+
+@click.command()
+@click.pass_context
+def main(ctx: click.Context) -> None:
+    """
+    Start the account-manager service as a foreground process.
+
+    This is a thin wrapper that defers the heavy import of server module.
```

Comment:
I think it doesn’t seem necessary to inform the user that this is a thin wrapper since that’s merely an implementation detail.